src/CAREUI/icons/CareIcon.tsx

```diff
@@ -50,6 +55,15 @@ export default function CareIcon({
         stroke={fill === false ? "currentColor" : undefined}
         strokeWidth={fill === false ? strokeWidth : undefined}
       />
+      {secondaryPath && (
+        <path
+          d={secondaryPath}
+          fill={fill === false ? "none" : "currentColor"}
+          stroke={fill === false ? "currentColor" : undefined}
+          strokeWidth={fill === false ? strokeWidth : undefined}
+          opacity={0.32}
+        />
+      )}
```

Comment:
we have a few duotone icons but many plain icons, so doing this condition check for all of them might be unnecessary
instead, can we make duotone icons a separate component?

Reply:
It is there for future proofing. Duotone icons might depend on these conditions

Reply:
An svg can have multiple paths right? So why don't we allow an array of paths instead?

Reply:
@rithviknishad we would need to change our existing icon schema for that. I am up for it, but thought its best to leave it be for the current scope